src/jemalloc.c

```diff
@@ -1420,6 +1422,9 @@ malloc_conf_init_helper(sc_data_t *sc_data, unsigned bin_shard_sizes[SC_NBINS],
 			CONF_HANDLE_UNSIGNED(opt_lg_tcache_flush_large_div,
 			    "lg_tcache_flush_large_div", 1, 16,
 			    CONF_CHECK_MIN, CONF_CHECK_MAX, /* clip */ true)
+			CONF_HANDLE_UNSIGNED(opt_debug_dblfree_cbin_scan_limit,
+			    "debug_dblfree_cbin_scan_limit", 0, 1024,
+                            CONF_DONT_CHECK_MIN, CONF_CHECK_MAX, /* clip */ true)
```

Comment:
No need to really limit the max, i.e. set it to UINT_MAX, and CONF_DONT_CHECK_MAX

If the user wants to set a max value to scan as many as possible, let's allow it.

Reply:
addressed in [5a2502a](https://github.com/jemalloc/jemalloc/pull/2310/commits/5a2502a2dd9fb3e401373e087983f334ed35dbc4)